### PR TITLE
Added glossary entries to Glossary page (2nd attempt)

### PR DESCRIPTION
**Pull request by scott-klaytn**
## Proposed changes

- Added glossary entries to Glossary page

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Minor Issues and Typos
- [x] Major Content Contribution
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to reach out. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn-docs/blob/master/CONTRIBUTING.md)
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn-docs)
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

- I'v made this PR for the second time because the original PR (https://github.com/klaytn/klaytn-docs/pull/592) has a problem with CLA signing due to incorrect email configuration in local repo.

### CODE REVIEW

**Review by ayo-klaytn — APPROVED** [review]

**Comment by scott-klaytn** [comment]
recheck

**Comment by scott-klaytn** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by praveen-klaytn — APPROVED** [review]

File: docs/misc/glossary.md [hunk 1]
```diff
@@ -1,3 +1,118 @@
...
+This glossary provides explanation on technical terms specific to the Klaytn Blockchain. It stands as your goto for understanding numerous terms you will encounter while exploring Klaytn documentation, guides and tutorials.
+
+### anchor
+A cryptographically secure reference to data stored outside of a blockchain. Anchors can be used to link data from different systems together, or to provide tamper-proof evidence of the existence of data at a specific point in time.
+
+### anchoring
+The process of storing an anchor on a blockchain. This can be done by creating a transaction that contains the hash of the external data, or by creating a smart contract that references the external data (for example, [Anchoring in service chain](https://docs.klaytn.foundation/content/installation-guide/deployment/service-chain/references/anchoring)).
+
+### Baobab
+The public testnet of the Klaytn platform. It is used by developers to test and verify their applications before deploying them to the mainnet.
```
Comment (neoofklaytn) [inline docs/misc/glossary.md:12]: `The public testnet of the Klaytn platform. It is used by developers to test and verify their applications before deploying them to the Cypress mainnet.`

Would be nice to add "Cypress" next to mainnet as below includes the link to Cypress

**Review by neoofklaytn — COMMENTED** [review]
Only few minor comments thanks

**Comment by neoofklaytn** [comment]
Would be nice to add those comments to a future PR thanks

**Pull request merged**